### PR TITLE
Update entando-k8s-operator.v0.3.x.clusterserviceversion.yaml

### PR DESCRIPTION
**Pull request by es-entando**
Update high level description of the Entando Operator

### CODE REVIEW

File: manifests/entando-k8s-operator.v0.3.x.clusterserviceversion.yaml [hunk 1]
```diff
@@ -35,31 +35,28 @@ spec:
...
-    environments, Red Hat SSO servers that can then be used by subsequent
-    deployments as an OIDC provider,
-
-    EntandoApps are used to deploy the appropriate Wildfly or JBoss EAP
-    containers hosting an Entando App, or potentially also custom containers.
-
-    EntandoPlugins can be used to deploy Entando microservice plugins in the
-    cluster and then link them to one or more EntandoApps.
+    **EntandoPlugins** for deploying microservices to customize or enhance your
+    EntandoApp. Entando microservice plugins can be deployed to your cluster, and
```
Comment (joewhite101) [inline manifests/entando-k8s-operator.v0.3.x.clusterserviceversion.yaml]: `can be` --> `are`

**Review by joewhite101 — APPROVED** [review]
Looks good overall. one small note. Ultimately we'll need to merge this to the main branch rather than the local-dev branch you chose as a target but we can do that in a separate step.

File: manifests/entando-k8s-operator.v0.3.x.clusterserviceversion.yaml (hunk 1 above)
Comment (es-entando) [reply manifests/entando-k8s-operator.v0.3.x.clusterserviceversion.yaml]: Updated text

**Comment by es-entando** [comment]
Thanks for the review and pointer on committing to the main branch. Can you let me know what the next steps are? Should I wait for this pull request to be merged? Do I make another pull request to the main branch?

**Pull request merged**